test/shared.integration.js

```diff
@@ -13,7 +13,7 @@ const SharedIntegration = require('./util/shared-integration.js');
 const SurveySuperTest = require('./util/survey-super-test');
 
 const models = require('../models');
-const config = require('../config');
+const config = require('../config/config');
```

Comment:
```
$ yarn test

Error: Cannot find module '../config/config'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:580:15)
    at Function.Module._load (internal/modules/cjs/loader.js:506:25)
    at Module.require (internal/modules/cjs/loader.js:636:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/home/work/git/amida-tech/amida-survey-microservice/test/shared.integration.js:16:16)
```

Reply:
Test (5464) pass for both `const config = require('../config');` and `const config = require('../config/test');`